workbench-core/example/infrastructure/src/__tests__/__snapshots__/example-stack.test.ts.snap

```diff
@@ -0,0 +1,1928 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TestExampleStack Infrastructure Test 1`] = `
```

Comment:
I'm pretty sure this test file was deleted in develop. @insignias can you confirm?

Reply:
Yes this file was deleted !